.examples/docker-compose/insecure/mariadb-cron-redis/apache/docker-compose.yml

```diff
@@ -25,6 +25,7 @@ services:
       - nextcloud:/var/www/html
     environment:
       - MYSQL_HOST=db
+      - REDIS_HOST=redis
```

Comment:
If the new config is used, the already existing one should be removed:
https://github.com/nextcloud/docker/blob/master/.examples/docker-compose/insecure/mariadb-cron-redis/apache/app/redis.config.php